### PR TITLE
[Hub Generated] Review request for Microsoft.Insights to add version stable/2018-03-01

### PR DESCRIPTION
**Pull request by yashmuel**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/yashmuel/azure-rest-api-specs/dev-monitor-Microsoft.Insights-2018-03-01-yashmuel...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json [hunk 1]
```diff
@@ -572,7 +637,7 @@
         },
         "properties": {
           "x-ms-client-flatten": true,
-          "$ref": "#/definitions/MetricAlertProperties",
+          "$ref": "#/definitions/MetricAlertPropertiesPatch",
```
Comment (MyronFanQiu) [inline specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json:640]: why do we change this one?

**Review by mmyyrroonn — CHANGES_REQUESTED** [review]
Please remove the submodule

File: specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json (hunk 1 above)
Comment (yashmuel) [reply specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json:640]: I've added MetricAlertPropertiesPatch object to be similiar to MetricAlertProperties, but not required any field, as it's a Patch request.

**Comment by MyronFanQiu** [comment]
@yashmuel Hi, please remove the sub module

**Comment by MyronFanQiu** [comment]
@yashmuel Hi, please remove the sub module

**Comment by MyronFanQiu** [comment]
@yashmuel.  Closed this PR since it's out of date. Feel free to open a new one.

**Pull request closed without merging**